tensorflow/dtensor/mlir/utils/collective_lowering.cc

```diff
@@ -95,6 +95,15 @@ bool HasEnableReuseGroupKey() {
   return true;
 }
 
+bool UseNcclCommunicationOnGpu() {
+  // FIXME(b/258703996): use tsl::ReadBoolFromEnvVar()
+  static const char* env_str = (std::getenv("DTENSOR_GPU_USE_NCCL_COMMUNICATION"));
+  if (env_str && strcmp(env_str, "0") == 0) {
```

Comment:
nit: Let's cross reference the definition in config.py.

Reply:
The Python side logic default is False. Let keep the default as False consistently in both Python and C++.